zebra-consensus/src/parameters/subsidy.rs

```diff
@@ -45,24 +45,34 @@ pub const POST_BLOSSOM_HALVING_INTERVAL: Height =
 /// Usage: founders_reward = block_subsidy / FOUNDERS_FRACTION_DIVISOR
 pub const FOUNDERS_FRACTION_DIVISOR: u64 = 5;
 
-/// The funding stream receiver categories
+/// The funding stream receiver categories.
 #[derive(Clone, Copy, Debug, Eq, Hash, PartialEq)]
 pub enum FundingStreamReceiver {
     Ecc,
     ZcashFoundation,
     MajorGrants,
 }
 
-/// Denominator as described in [protocol specification §7.9.1][7.9.1].
+impl FundingStreamReceiver {
+    /// Get a list of receiver types
+    pub fn receivers() -> Vec<Self> {
+        vec![Self::Ecc, Self::ZcashFoundation, Self::MajorGrants]
+    }
+}
+
+/// The number of funding stream receiver categories.
+pub const FUNDING_STREAM_RECEIVERS_NUMBER: usize = 3;
```

Comment:
Optional idea to reduce risk of future changes:

```suggestion
impl FundingStreamReceiver {
    /// Get a list of receiver types
    pub const fn receivers() -> [Self; 3] {
        [Self::Ecc, Self::ZcashFoundation, Self::MajorGrants]
    }
}

/// The number of funding stream receiver categories.
pub const FUNDING_STREAM_RECEIVERS_NUMBER: usize = FundingStreamReceiver::receivers().len();
```

Reply:
Thanks, i think this is a bit better. Done in https://github.com/ZcashFoundation/zebra/pull/3040/commits/3677de67c361a9388e9134b1cf749780501633d9